doc/ocpp_201_status.md

```diff
@@ -1243,8 +1243,8 @@ This document contains the status of which OCPP 2.0.1 numbered functional requir
 | K01.FR.11 |        |                                                                                                 |
 | K01.FR.12 |        |                                                                                                 |
 | K01.FR.13 |        |                                                                                                 |
-| K01.FR.14 |        |                                                                                                 |
-| K01.FR.15 |        |                                                                                                 |
+| K01.FR.14 | ✅     |                                                                                                 |
+| K01.FR.15 | ✅     |                                                                                                 |
```

Comment:
These are not addressed in this PR, right?

Reply:
These were addressed with the check [here](https://github.com/EVerest/libocpp/blob/b67951cebbff4de26456e4575e3ff6e84c4630ed/lib/ocpp/v201/smart_charging.cpp#L319)